### PR TITLE
fix flex #1532

### LINKED ISSUE #1532: remove redundancy api of Flex
https://github.com/ant-design/ant-design-mobile/blob/master/components/flex/Flex.web.tsx#L35

### PR DESCRIPTION
**Pull request by pingan1927**
close #1532 

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [x] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/1549)

### CODE REVIEW

**Comment by paranoidjk** [comment]
@pingan1927 两个问题，
- ci lint 挂了，本地注意重装依赖跑下保证没问题
- 同步更新 2.0 的 changlog，避免后面忘记

**Comment by pingan1927** [comment]
@paranoidjk 你仔细看看lint内容，目前的几个lint报错都是其他组件引起的。且都不是代码层面的问题。

**Comment by paranoidjk** [comment]
@pingan1927 rebase master, 我上周已经修复了。

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ timeline: true
   to `input`(for web) or `TextInput`(for react native) element;
 - Progress: `wrapStyle` has been replaced by `style` and  `style` has been replaced by `barStyle` property.
 - The `style` property now can custom style, eg: tintColor for switch
+- `Flex` 清理冗余API. [#1532](https://github.com/ant-design/ant-design-mobile/issues/1532)
```
Comment (paranoidjk) [inline CHANGELOG.en-US.md]: 写明去掉了哪些 api。

changlog 存在的意义是用户升级的时候一个速查表，而不是还要他点进去看 issue，看代码。

附上 issue 只是给有需要的人，不是所有人。

**Comment by paranoidjk** [comment]
close #1532  应该写在 PR 描述的开头。

review 的问题我帮你修复了。先合掉

**Pull request merged**